packages/react-ui/.storybook/preview-head.html

```diff
@@ -22,6 +22,9 @@
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
     text-rendering: optimizeLegibility;
+    @media (max-width: 600px) {
```

Comment:
Could we add global styles to react-ui and use globalStyle definition in `@cdt/frontend` ?

Reply:
That's a great idea 😲  Great great idea !

Reply:
Yes i already did it on a project

Reply:
Done, it works !